draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -2142,10 +2182,10 @@ meeting the requirements in {{rational-map}}.
 Helper functions:
 
 - map\_to\_curve\_elligator2 is the mapping of {{elligator2}} to the curve M.
-- rational\_map is a function that takes a point (x, y) on M and
+- rational\_map is a function that takes a point (X, Y) on M and
```

Comment:
Since elligator2 uses a Weierstrass curve defined above, then `M` (which previously referred to a Montgomery curve) got undefined.

Reply:
Yeah, I see what you mean. We could call it E', I suppose.

The fundamental issue is that, as you have pointed out above, the "Montgomery to Edwards" section is inconsistent. Let's fix this after discussing in #195 rather than here, OK?